src/components/HistoryTable/row/FoldedCommit.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { FoldedCommit } from "./FoldedCommit";
+
+export default {
+  title: "Folded Commit",
+};
+
+export const FoldedInactiveCommits = () => (
+  <FoldedCommit rolledUpCommits={rolledUpCommits} />
+);
```

Comment:
Can we include these stories in the HistoryTable Stories. Like we do for the icons.